cluster/local/integration_tests.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+set -e
+
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+projectdir="${scriptdir}/../.."
+
+# get the build environment variables from the special build.vars target in the main makefile
+eval $(make --no-print-directory -C ${projectdir} build.vars)
+BUILD_IMAGE="${BUILD_REGISTRY}/${PROJECT_NAME}-amd64"
+
+helm_tag="$(cat ${projectdir}/_output/version)"
+CROSSPLANE_IMAGE="${DOCKER_REGISTRY}/${PROJECT_NAME}:${helm_tag}"
+K8S_CLUSTER="${BUILD_REGISTRY}-INTTESTS"
+
+CROSSPLANE_NAMESPACE="crossplane-system"
+
+# cleanup on exit
+function cleanup {
+    export KUBECONFIG=
+    "${KIND}" delete cluster --name="${K8S_CLUSTER}"
+}
+trap cleanup EXIT
+
+# create cluster
+"${KIND}" create cluster --name="${K8S_CLUSTER}"
+export KUBECONFIG="$("${KIND}" get kubeconfig-path --name="${K8S_CLUSTER}")"
+
+# tag crossplane image and load it to kind cluster
+docker tag "${BUILD_IMAGE}" "${CROSSPLANE_IMAGE}"
+"${KIND}" load docker-image "${CROSSPLANE_IMAGE}" --name="${K8S_CLUSTER}"
+
+# install tiller
+"${KUBECTL}" apply -f "${scriptdir}/helm-rbac.yaml"
+"${HELM}" init --service-account tiller
+# waiting for deployment "tiller-deploy" rollout to finish
+"${KUBECTL}" -n kube-system rollout status deploy/tiller-deploy --timeout=2m
+
+# install crossplane
+echo "installing helm package(s) into \"$ns\" namespace"
+"${HELM}" install --name "${PROJECT_NAME}" --namespace "${CROSSPLANE_NAMESPACE}" "${projectdir}/cluster/charts/${PROJECT_NAME}" --set image.pullPolicy=Never,imagePullSecrets=''
+
+# waiting for deployment "crossplane" rollout to finish
+"${KUBECTL}" -n "${CROSSPLANE_NAMESPACE}" rollout status "deploy/${PROJECT_NAME}" --timeout=2m
+echo "wait for 5 seconds so that the pods are up and running"
+sleep 5
+
+# ----------- integration tests
+# get the pods statuses
+podsstatuses=`"${KUBECTL}" -n "${CROSSPLANE_NAMESPACE}" get pods -o=jsonpath='{range .items[*]}{@.metadata.name}{":"}{@.status.phase}{" "}'`
+
+# check for minimum number of pods created
+MIN_CROSSPLANE_PODS=2
+IFS=' ' read -ra PODS_ARRAY <<< "$podsstatuses"
```

Comment:
What is the reason setting the `IFS` is needed? If I remember correctly, the default value should be whitespace already

Reply:
just being consistent with the IFS used a few line later.